modules/dailymotionBidAdapter.js

```diff
@@ -136,9 +136,21 @@ export const spec = {
         bidId: bid.bidId || '',
         mediaTypes: {
           video: {
-            playerSize: bid.mediaTypes?.[VIDEO]?.playerSize || [],
             api: bid.mediaTypes?.[VIDEO]?.api || [],
-            startDelay: bid.mediaTypes?.[VIDEO]?.startdelay || 0,
+            ext: bid.mediaTypes?.[VIDEO]?.ext || {},
+            maxseq: bid.mediaTypes?.[VIDEO]?.maxseq || 0,
```

Comment:
I don't see `ext` & `maxseq` in https://docs.prebid.org/dev-docs/adunit-reference.html#video